app/templates/index.hbs

```diff
@@ -17,10 +17,17 @@
  {{!--  TURN THIS OFF TO GET RID OF THE MAP  --}}
 <div class="map-container">
   {{#if this.selection.getEntireGeoTask.isRunning}}
-    <div class="spinner">
-      <div class="bounce"></div>
-      <div class="bounce"></div>
-      <div class="bounce"></div>
+    <div class="loading-wrapper">
```

Comment:
Can you make this into an ember component? I know the existing code already had the spinner code duplicated in multiple places but this seems like a pretty easier opportunity to leave something nicer than we found it. The rest of the PR lgtm

Reply:
🤦  Why didn't I think of it?